resources/dictionary.txt

```diff
@@ -51,3 +51,184 @@ SOP
 underserved
 VICC
 www
+ACMG
+AKT
+Albracht
+BAC
+Barnell
+Bouk
+Coffman
+Danos
+EGFR
+FFPE
+GRC
+GRCh
+Grisdale
+HGVS
+Hanahan
+Hypomorphic
+Hypermorphic
+IGV
+KRAS
+Kitts
+Kiwala
+Krysiak
+Kujan
+LOF
+McMichael
+NGS
+Neomorphic
+Nissen
+Oncogenic
+Pema
+Ph
+PubMed
+Raca
+Rao
+Ritter
+SBS
+Sheta
+Sonkin
+Thibaud
+Wollam
+actionability
+al
+allelic
+bp
+colorectal
+counterindication
+de
+et
+germline
+hg
+hypomorphic
+kb
+kilobase
+kinases
+novo
+nucleotides
+oncogenic
+phenotypes
+segmentally
+siloed
+subtypes
+tyrosine
+vitro
+word
+ACMG
+AKT
+Albracht
+BAC
+Barnell
+Bouk
+Coffman
+Danos
+EGFR
+FFPE
+GRC
+GRCh
+Grisdale
+HGVS
+Hanahan
+Hyopmorphic
+Hypermorphic
+IGV
+KRAS
+Kitts
+Kiwala
+Krysiak
+Kujan
+LOF
+McMichael
+NGS
+Neomorphic
+Nissen
+Oncogenic
+Pema
+Ph
+PubMed
+Raca
+Rao
+Ritter
+SBS
+Sheta
+Sonkin
+Thibaud
+Wollam
+actionability
+al
+allelic
+bp
+colorectal
+counterindication
+de
+et
+germline
+hg
+hypomorphic
+independant
+kb
+kilobase
+kinases
+novo
+nucleotides
+oncogenic
+phenotypes
+segmentally
+siloed
+subtypes
+theeffect
+tyrosine
+vitro
+word
+AACR
+ABL
+BCR
+CCG
+CML
+CNLOH
+CNVs
+Catalogue
+ESMO
+FGFR
+ICGC
+Indels
+KMS
+MSK
+Missense
+NTRK
+OnkoKB
+Parimony
+Proline
+SNVs
+Submitters
+TCGA
+TGA
+TGC
+Translocations
+VCEP
+VCEPs
+VCF
+VUS
+actionablilty
+afatinib
+allignment
+amorphic
+bioinformatics
+biomarker
+disregulated
+duplications
+erlotinib
+frameshift
+imatinib
+immunotherapy
+indels
+missense
+myeloid
+preclinical
+submitters
+targetable
+tetramer
+th
+tiering
+word
```

Comment:
Looks like this file has a bunch of duplicate entries now.

Reply:
I also uniq'd it.